packages/integration-tests/src/components/events/test-mouseover/integration/mouseover.spec.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const assert = require('assert');
+const URL = '/mouseover';
+
+describe('mouseover', () => {
+    beforeEach(async () => {
+        await browser.url(URL);
+    });
+
+    it('should be able to trigger programmatic mouseover', async () => {
+        const component = await browser.$('integration-mouseover');
+        const elementToHover = await component.shadow$('.mouseover');
+        // This might seem like a roundabout way to mouse over an element, but this puts ChromeDriver into the
+        // code path where it calls elementsFromPoint, which is what we're trying to test:
+        // https://github.com/bayandin/chromedriver/blob/ad6ede8/js/get_element_location.js#L122
```

Comment:
I was able to hit this code path by using: [`dragAndDrop`](https://webdriver.io/docs/api/element/dragAndDrop)

```js
const component = await browser.$('integration-mouseover');
const elementToHover = await component.shadow$('.mouseover');

await elementToHover.dragAndDrop({ x: 100, y: 200 })
```

Reply:
Ah nice! I couldn't seem to find any succinct API in WebDriverIO to hit this. (I looked in the Python test's logs and reverse-engineered it.) I'll use `dragAndDrop`.

Reply:
Actually I think I'd prefer to keep the explicit `performActions`. Reasons:

- `dragAndDrop` [just uses `performActions()` under the hood](https://github.com/webdriverio/webdriverio/blob/9d2220e89144b0ca69232737957ad5fc32ca1300/packages/webdriverio/src/commands/element/dragAndDrop.ts#L112-L123)
- if WebDriverIO changes their implementation, then the test may no longer actually hit `elementsFromPoint`
- the test would fail in IE (I believe) because `performActions` is not supported
- I'd have to write a more complex drag-and-drop scenario to properly test it

OTOH I realized I could simplify the `performActions` code so it's not so verbose.